source/simulator/core.cc

```diff
@@ -2153,6 +2167,13 @@ namespace aspect
 
               pcout << std::endl;
 
+              // output relative residuals
```

Comment:
Why does `iterated_IMPES` also output the absolute residuals? The other schemes do not do this. Do we want to keep this behavior? Then ignore my two comments above.